contributors/devel/sig-node/updating-pause-images.md

```diff
@@ -0,0 +1,30 @@
+# Updating Pause Image References
+
+The Kubernetes `pause` image that is used to run a 'parent' container for all
```

Comment:
```suggestion
The Kubernetes `pause` image is used to run a 'parent' container for all
```

should we mention sandbox here?

Reply:
I don't think it's worth it - linked docs about it tho